### ISSUE TITLE
Errors with sox 14.4.2  [Newer dependencies]

### ISSUE THREAD

**Issue by ycwan** [m-ab-s/media-autobuild_suite#1431]
``` bash
14:16:01 sox 14.4.2  ............................... [Newer dependencies]
14:16:06 ┌ sox-14.4.2  ..................................... [Downloaded]
14:16:13 Installing libmad... done
14:16:21 ├ Running uninstall...
14:16:24 Installing opusfile... done
14:16:32 ├ Running configure...
14:19:17 ├ Running make...
Likely error (tail of the failed operation logfile):
C:/mingw/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: ./.libs/libsox.a(libsox_la-flac.o):flac.c:(.text+0xb77): undefined reference to `__imp_FLAC__stream_decoder_process_until_end_of_metadata'
C:/mingw/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: ./.libs/libsox.a(libsox_la-flac.o):flac.c:(.text+0xb85): undefined reference to `__imp_FLAC__stream_decoder_get_state'
C:/mingw/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: ./.libs/libsox.a(libsox_la-flac.o):flac.c:(.text+0xcf9): undefined reference to `__imp_FLAC__stream_encoder_get_state'
C:/mingw/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: ./.libs/libsox.a(libsox_la-flac.o):flac.c:(.text+0xd05): undefined reference to `__imp_FLAC__stream_encoder_finish'
C:/mingw/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: ./.libs/libsox.a(libsox_la-flac.o):flac.c:(.text+0xd0f): undefined reference to `__imp_FLAC__stream_encoder_delete'
C:/mingw/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/9.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: ./.libs/libsox.a(libsox_la-flac.o):flac.c:(.text+0xd21): undefined reference to `__imp_FLAC__metadata_object_delete'
collect2.exe: error: ld returned 1 exit status
make[1]: *** [Makefile:1410: sox.exe] Error 1
make[1]: Leaving directory '/build/sox-14.4.2/build-64bit/src'
make: *** [Makefile:644: all-recursive] Error 1
make failed. Check C:/mingw/build/sox-14.4.2/build-64bit/ab-suite.make.log
This is required for other packages, so this script will exit.
14:20:19   Creating diagnostics file...

All relevant logs have been anonymously uploaded to https://0x0.st/zxah.zip
Copy and paste [logs.zip](https://0x0.st/zxah.zip) in the GitHub issue.
Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.
Try running the build again at a later time.
```

[logs.zip](https://0x0.st/zxah.zip)
[ab-suite.make.log](https://github.com/m-ab-s/media-autobuild_suite/files/3721507/ab-suite.make.log)

**Comment by BlackScreen** [comment]
It is still an issue - see my post in #1434.

**Comment by ycwan** [comment]
BlackScreen, I think the closed issue #1426 have the answer for the issue you are facing

**Comment by BlackScreen** [comment]
Deleting the flac build folder seems to have fixed this. Thanks for the tip and sorry for the issue.
